### PR TITLE
GET requests completed with fake data

### PR DESCRIPTION
**Pull request by sfshaider**
Completed testing of get request with dummy faker data. Working on seed.

### CODE REVIEW

File: database/models.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    Game.find({}).exec( (err, res) => {
+      if(err){
+        console.log("error in getAll")
+        throw err;
+      } else {
+        callback(res);
+        console.log("getAll success")
+      }
+    })
+  };
```
Comment (BHamp) [inline database/models.js:61]: Is there a reason you have a get all function and not one for a solitary query?

File: database/models.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    var example = new Game ({
+        name: faker.name.findName(),
+        email: faker.internet.email(),
+        address: faker.address.streetAddress(),
+        bio: faker.lorem.sentence(),
+     })
+    example.save(err => { 
+      if (err) return console.log(err) });
+    }
+  }
```
Comment (BHamp) [inline database/models.js:36]: I'm not sure if this is appropriate separation of concerns. I don't know if you should have your data generator in the models file

File: database/models.js (hunk 1 above)
Comment (sfshaider) [reply database/models.js:61]: This was part of the basic set up to ensure there was communication with the database. I am implement getOne with next commit.

File: database/models.js (hunk 2 above)
Comment (sfshaider) [reply database/models.js:36]: Part of testing. Will be moved to separate seed file.

**Pull request merged**